test/suite/test_tiered10.py

```diff
@@ -29,6 +29,7 @@
 from helper_tiered import generate_s3_prefix, get_auth_token, get_bucket1_name
 from wtscenario import make_scenarios
 import os, wiredtiger, wttest
+import time
```

Comment:
do we need this import?

Reply:
Not needed, that was used for debugging.